### DISCUSSION TITLE
PodcastGenerator website?

### DISCUSSION THREAD

**Discussion by gmhall506** [PodcastGenerator/PodcastGenerator#722] [General]
Does anyone know where PodcastGenerator now lives on the web? podcastgenerator.net is no longer.

**Comment by charabaruk** [comment]
PodcastGenerator.net will return, but for now the GitHub project is it. I need to make arrangements with Alberto (the guy who originally started PG) but I've been quite busy with other things and haven't had an opportunity to take over running PodcastGenerator.net.

For now, podcastgenerator.github.io can be relied on but it is itself quite out of date unfortunately.

**Comment by gmhall506** [comment]
Thanks!
______________
Geoffrey Hall
95 Lexington Lane, Fredericton  NB  E3A 5S6
***@***.***
(506) 454 4401 [h]  (506) 440 1115 [m]  (506) 450-7761 [o]
